### ISSUE TITLE
Simler Kusto Cli Release Model

### ISSUE THREAD

**Issue by yogilad** [Azure/azure-kusto-dotnet#36]
1. New GitHub Repo for Kusto CLI releases with basic Readme, license and **No** source code
2. Compile single process apps and upload to github releases 
3. Compile and deploy dotnet tool
4. Add repo to this board

**Comment by Arithmomaniac** [comment]
Yes. Still waiting for a `DotnetToolSettings.xml` file so we can install with `dotnet tool`.
